docs/api/info.md

```diff
@@ -284,21 +284,71 @@ $FP(callsign):SERVER:(VFR/IFR):(acft):(spd):(origin):(sched dep):1000:(crz alt):
 VATSIM's flightplan format is simpler, and is based on the [FAA domestic flight plan [PDF]](https://www.faa.gov/documentLibrary/media/Form/FAA_Form_7233-1_7_31_17.pdf).
 
 ```
-$FP(callsign):*A:(VFR/IFR):(acft type):(spd):(origin):(sched dep):(sched dep):(alt):(dest):(EET):(FOB):(alt airport):(remarks)
+$FP(callsign):*A:(VFR/IFR):(acft type):(spd):(origin):(sched dep):(sched dep):(alt):(dest):(EET):(FOB):(alt airport):(remarks):(route)
```

Comment:
Is `(route)` just the filed route? e.g. `SID WAYPT A123 WAYPT STAR`

Reply:
Yes

Reply:
Yes